config/v1alpha1/types_cluster_monitoring.go

```diff
@@ -77,6 +79,13 @@ type ClusterMonitoringSpec struct {
 	// userDefined set the deployment mode for user-defined monitoring in addition to the default platform monitoring.
 	// +required
 	UserDefined UserDefinedMonitoring `json:"userDefined"`
+
+	// metricsServer defines the configuration for the Metrics Server component.
+	// The Metrics Server provides container resource metrics for use in autoscaling pipelines.
```

Comment:
While I understood what most of this jargon meant, if you are looking to explain what the metrics server does, I would recommend being a bit more verbose. From the metrics-server doc site, this is a pretty good and succinct summary IMO:

>Metrics Server collects resource metrics from Kubelets and exposes them in Kubernetes apiserver through Metrics API for use by Horizontal Pod Autoscaler and Vertical Pod Autoscaler.

This gives me, as a user, a bit more context as to why I might care about doing some configuration of this field